seqr/views/utils/orm_to_json_utils.py

```diff
@@ -681,6 +685,15 @@ def get_project_collaborators_by_username(project, include_permissions=True):
     """Returns a JSON representation of the collaborators in the given project"""
     collaborators = {}
 
+    acl = anvilSessionStore.get_session().get_workspace_acl(project.workspace_namespace, project.workspace_name)
```

Comment:
`anvilSessionStore.get_session()` without a `user` as parameter means getting the service account session. It must be successful.

Reply:
but what if the user is logged in with local seqr and not AnVIL and shouldn't have access to terra at all?

Reply:
Changed to call get_anvil_workspace_acl() directly according to the comments below.

Reply:
Since we don't have a user object passed into this function, we can't know if the user signed in with AnVIL or not. And I don't think it is a problem if a user logged in locally knows there are some users from AnVIL can access the project.